cmd/createcluster_internal_test.go

```diff
@@ -108,6 +136,7 @@ func testCreateCluster(t *testing.T, conf clusterConfig) {
 
 	dir, err := os.MkdirTemp("", "")
 	require.NoError(t, err)
+	defer os.RemoveAll(dir)
```

Comment:
no need to delete temp dirs, you might not have permissions either

Reply:
yeah there maybe a permissions issue, good point